Makefile

```diff
@@ -68,7 +72,7 @@ real-clean: clean
 # Python environment and dependencies
 venv: $(VENVPATH)
 $(VENVPATH):
-	unset PYTHONPATH; virtualenv -p$(PYTHON) $(VENVPATH)
+	unset PYTHONPATH; $(VIRTUALENV) $(VENVPATH)
```

Comment:
FWIW, the `unset PYTHONPATH` is a remnant of less enlightened times, and can be removed.